pipelines/build/regeneration/build_pipeline_generator.groovy

```diff
@@ -154,8 +154,9 @@ node('master') {
           BRANCH              : repoBranch,
           BUILD_FOLDER        : jobRoot,
           CHECKOUT_CREDENTIALS: checkoutCreds,
+          JAVA_VERSION        : javaVersion,
           JOB_NAME            : "openjdk${javaVersion}-pipeline",
-          SCRIPT              : "${scriptFolderPath}/openjdk${javaVersion}_pipeline.groovy",
+          SCRIPT              : "${scriptFolderPath}/openjdk_pipeline.groovy",
```

Comment:
There's a [few places](https://github.com/AdoptOpenJDK/ci-jenkins-pipelines/search?q=pipeline) where comments and/or code may need to be updated

Reply:
updated the docs